actions/bind

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configuration helper for BIND server.
+"""
+
+import argparse
+
+from plinth import action_utils
+
+
+CONFIG_FILE = '/etc/bind/named.conf.options'
+CONFIG_FILE_2 = '/etc/bind/named_temp.conf.options'
```

Comment:
We could avoid the temp file, with a procedure like this:
1. Open config file in read mode.
2. Read all the lines into a list.
3. Close the config file.
4. Open config file in write mode.
5. Loop through lines, make changes, and write to file.
6. Close the config file.
(We can also skip the close steps if we use with-statement contexts.)